test/suites/migration.sh

```diff
@@ -423,8 +429,14 @@ migration() {
   lxc_remote storage volume snapshot l1:"$remote_pool1" vol3
   lxc_remote storage volume set l1:"$remote_pool1" vol3 user.foo=postsnap1vol3
 
-  # check snapshot volumes and snapshots are refreshed
+  # check that a refresh doesn't change the volume's but snapshot's UUID.
+  old_uuid="$(lxc storage volume get l2:"$remote_pool2" vol2 volatile.uuid)"
+  old_snap0_uuid="$(lxc storage volume get l2:"$remote_pool2" vol2/snap0 volatile.uuid)"
   lxc_remote storage volume copy l1:"$remote_pool1/vol3" l2:"$remote_pool2/vol2" --refresh
+  [ "$(lxc storage volume get l2:"$remote_pool2" vol2 volatile.uuid)" = "${old_uuid}" ]
+  [ "$(lxc storage volume get l2:"$remote_pool2" vol2/snap0 volatile.uuid)" != "${old_snap0_uuid}" ] # Custom volume snapshots always get recreated.
```

Comment:
Can you clarify the comment here, the snapshots always get recreated? Or the snapshot's uuid always gets recreated? Both sound not quite right to me.

Reply:
Changed based on the recent fix.